test/fixture-auto-retry/custom-errors/script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+FILENAME=/tmp/$1
```

Comment:
NIT: we mostly follow the [Google shell style guide](https://google.github.io/styleguide/shellguide.html). Same for a few other suggestions below.

```suggestion
FILENAME="/tmp/$1"
```